apps/passport-server/src/services/issuanceService.ts

```diff
@@ -887,14 +1067,133 @@ export class IssuanceService {
       return tickets;
     });
   }
+
+  /**
+   * Verifies a ticket based on:
+   * 1) verification of the PCD (that it is correctly formed, with a proof
+   *    matching the claim)
+   * 2) whether the ticket matches the ticket types known to us, e.g. Zuzalu
+   *    or Zuconnect tickets
+   *
+   * Not used for Devconnect tickets, which have a separate check-in flow.
+   */
+  private async verifyTicket(
+    serializedPCD: SerializedPCD
+  ): Promise<VerifyTicketResult> {
+    if (!serializedPCD.type) {
+      throw new Error("input was not a serialized PCD");
+    }
+
+    if (serializedPCD.type !== EdDSATicketPCDPackage.name) {
+      throw new Error(
+        `serialized PCD was wrong type, '${serializedPCD.type}' instead of '${EdDSATicketPCDPackage.name}'`
+      );
+    }
+
+    await EdDSAPCDPackage.init?.({});
+    await EdDSATicketPCDPackage.init?.({});
```

Comment:
I think this is the source of the bug that you experienced in `instrumentPCDs`. `EdDSAPCDPackage` doesn't have an `init` function. If you remove this line of code, the error you outlined in `instrumentPCDs` will go away. Separately, I should make my instrumentation code not instrument functions that don't exist, so that the error you get when you try to invoke one that is instrumented is clearer. https://github.com/proofcarryingdata/zupass/issues/827